topology/sof/pipe-eq-fir-volume-playback.m4

```diff
@@ -52,7 +52,7 @@ C_CONTROLBYTES(EQFIR, PIPELINE_ID,
 W_PCM_PLAYBACK(PCM_ID, Passthrough Playback, 2, 0, 2)
 
 # "Volume" has 2 source and 2 sink periods
-W_PGA(0, PIPELINE_FORMAT, 2, 2, 2, LIST(`		', "Master Playback Volume PIPELINE_ID"))
+W_PGA(0, PIPELINE_FORMAT, 2, 2, 2, LIST(`		', "PIPELINE_ID Master Playback Volume"))
```

Comment:
This should be fixed in alsa-mixer. Kcontrol naming does follow the ALSA SOURCE:DIRECTION:FUNCTION scheme but we have appended an ID (pipeline ID). i.e. SOURCE:DIRECTION:FUNCTION :<OTHER DETAIL>

Reply:
@lgirdwood We appended <OTHER DETAIL>, that is why alsamixer can 't separate playback and capture, how about make `SOURCE:<OTHER DETAIL>:DIRECTION:FUNCTION` ? this way the alsamixer will work properly.

Reply:
@plbossart @ranj063 @bardliao 
Could you share your thoughts on this? 
1. If don't change alsamixer, we need to change the kcontrol name as` SOURCE:<OTHER DETAIL>:DIRECTION:FUNCTION`, because the last two DIRECTION:FUNCTION are used for classify in alsamixer, and DIRECTION:FUNCTION will not display in item name, only SOURCE:<OTHER DETAIL> will.
2. If change alsamixer, I can make a patch, but I don't know if we have enough reason to do that.

Reply:
@zhuyingjiang ok, lets go with 
```
SOURCE <OTHER DETAIL>:DIRECTION:FUNCTION 
```
Note the space between SOURCE and <OTHER DETAIL>. This will change the kcontrol naming meaning UCM files will need updated too. @cujomalainey do you have a link where @zhuyingjiang can update the UCM file to match the new name ?

Reply:
@lgirdwood I'm not sure we set any fw kcontrols in the ucm file today. @cujomalainey can you please confirm?

Reply:
@zhuyingjiang this PR should now also be merged into SOF repo

Reply:
@ranj063 I checked the UCM files from the release. No sign of any PGA controls.

Reply:
@lgirdwood if this will not effect on UCM, I think we can merge it.

Reply:
@zhuyingjiang ok, can you send to SOF repo, this repo is deprecated now.

Reply:
OK